scripts/globals/items/bottle_of_hysteroanima.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------------
+-- ID: 5262
+-- Hysteroanima
+-- Using a Hysteroanima on an Empty foe in Promyvion will prevent it from using any TP abilities for 30 seconds, similar to an Amnesia effect
+-- Abilities will appear to be readied, but will not execute. Regular attacks may still be used. 
+-----------------------------------------
+require("scripts/globals/msg")
+require("scripts/globals/status")
+require("scripts/globals/zone")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = dsp.msg.basic.ITEM_UNABLE_TO_USE -- default result is fail
+    local family = target:getSystem()
+    local zone = target:getZoneID()
+    local validZoneList =
+            {
+                16,17,18,19,20,21,22,23 -- only usable in Promyvion areas
```

Comment:
does it work on just any mob within these zones? honestly I never tried it against non bosses

Reply:
also your indent is wrong

Reply:
Thats what i was thinking. Be funny watching the memory recepticle dance around

Reply:
Wiki says on any Empty enemies in Promyvion zones which is exactly what the check does, and Wiggo's link shows it working on a Wanderer. Unsure of behavior against MR, but in that case the fix might be changing its systemid to something so it's not categorized as an Empty.